### ISSUE TITLE
Redirect the old people URLs

### ISSUE THREAD

**Issue by amirmc** [ocamllabs/ocl-reporter#49]
URLs of the individual people pages — of the form `http://www.cl.cam.ac.uk/projects/ocamllabs/people/<person-id>.html` — should be redirected to the group people page at http://www.cl.cam.ac.uk/projects/ocamllabs/people/index.html

Search engines have already indexed the site and looking for people in OCL currently leads to broken links. We could do this in the site by making redirect pages, which is something of crude solution but it does let crawlers update themselves.

**Comment by avsm** [comment]
This risks keeping it live for quite some time.  A site.xml might be a quicker way to trigger a cleanup of the search engines (I'm surprised that the 404s haven't caused it to be removed from Google already)

**Comment by amirmc** [comment]
A site.xml and explicitly triggering Google to index it (via webmaster tools?) would probably be a cleaner solution.  I doubt many people have linked directly to those pages (I certainly haven't been).  It's probably A Good Thing to generate such a file on every build anyway.

**Comment by yallop** [comment]
People have sometimes linked directly to my people page in lieu of a homepage.

**Comment by avsm** [comment]
Ok, in that case I'll bring them back with a redirect to your registered homepage (which could just be GitHub).  It wasn't really the intention to use this URL space as proper homepages, as they're not representative of the person, just a collection of links to projects.

**Comment by amirmc** [comment]
Yeah, but in lieu of an *actual* personal webpage, people will link to whatever looks like it has the most info.  I typically link to GitHub profiles but it shouldn't be surprising that others may have liked to Jeremy's OCL page.
